demisto_sdk/commands/common/errors.py

```diff
@@ -2149,3 +2150,8 @@ def changed_pack_name(original_name):
     @error_code_decorator
     def wrong_version_format():
         return 'Pack metadata version format is not valid. Please fill in a valid format (example: 0.0.0)'
+
+    @staticmethod
+    @error_code_decorator
+    def missing_readme_file(location):
+        return f'{location} is missing a readme file'
```

Comment:
```suggestion
        return f'{location} must have a README.md file.'
```

Reply:
didn't want to write it this way because this is not necessarily the name of the file.

Reply:
Done